src/main/java/net/helpscout/api/utils/ParamsUtils.java

```diff
@@ -0,0 +1,84 @@
+package net.helpscout.api.utils;
+
+import lombok.SneakyThrows;
+
+import java.net.URLEncoder;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @Author: ivan
+ * Date: 28.04.16
+ * Time: 0:29
+ */
+public class ParamsUtils {
+
+    @SneakyThrows
+    public static String setParams(String url, Map<String, String> params) {
+
+        if (params != null && params.size() > 0) {
+            StringBuilder sb = new StringBuilder();
+            sb.append(url);
+            for (String key : params.keySet()) {
+                appendParamSeparator(sb);
+                String encodedParameter = URLEncoder.encode(params.get(key), "UTF-8");
+                sb.append(key).append("=").append(encodedParameter);
+            }
+            return sb.toString();
+        }
+        return url;
+    }
+
+    public static String setFields(String url, List<String> fields) {
+
+        if (fields != null && fields.size() > 0) {
+            StringBuilder sb = new StringBuilder();
+            sb.append(url);
+            appendParamSeparator(sb);
+            sb.append("fields=");
+
+            String sep = "";
+            for (String field : fields) {
```

Comment:
https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/StringUtils.html#join(java.util.Collection, char) could be more readable